spec/features/polls/polls_spec.rb

```diff
@@ -193,5 +193,20 @@
       expect(page).to have_link('Chewbacca')
     end
 
+    scenario 'User can write comments' do
+      create(:comment, commentable: poll)
+
+      visit poll_path(poll)
+
+      expect(page).to have_css('.comment', count: 1)
+
+      comment = Comment.last
+      within first('.comment') do
+        expect(page).to have_content comment.user.name
+        expect(page).to have_content I18n.l(comment.created_at, format: :datetime)
+        expect(page).to have_content comment.body
+      end
+    end
+
```

Comment:
All comment scenarios should be tested https://github.com/consul/consul/tree/master/spec/features/comments

Reply:
Either add to the Warning section of this PR that only the basic scenarios have been tested 
or test all the scenarios that a comment can have https://github.com/consul/consul/blob/master/spec/features/comments/proposals_spec.rb